third_party/bazel/extra_actions_base_pb2.py

```diff
@@ -75,12 +85,52 @@
   ],
   options=None,
   is_extendable=False,
+  syntax='proto2',
   extension_ranges=[],
+  oneofs=[
+  ],
   serialized_start=123,
   serialized_end=213,
 )
 
 
+_EXTRAACTIONINFO_ASPECTPARAMETERSENTRY = _descriptor.Descriptor(
+  name='AspectParametersEntry',
+  full_name='blaze.ExtraActionInfo.AspectParametersEntry',
+  filename=None,
+  file=DESCRIPTOR,
+  containing_type=None,
+  fields=[
+    _descriptor.FieldDescriptor(
+      name='key', full_name='blaze.ExtraActionInfo.AspectParametersEntry.key', index=0,
+      number=1, type=9, cpp_type=9, label=1,
+      has_default_value=False, default_value=_b("").decode('utf-8'),
+      message_type=None, enum_type=None, containing_type=None,
+      is_extension=False, extension_scope=None,
+      options=None),
+    _descriptor.FieldDescriptor(
+      name='value', full_name='blaze.ExtraActionInfo.AspectParametersEntry.value', index=1,
+      number=2, type=11, cpp_type=10, label=1,
+      has_default_value=False, default_value=None,
+      message_type=None, enum_type=None, containing_type=None,
+      is_extension=False, extension_scope=None,
+      options=None),
+  ],
+  extensions=[
+  ],
+  nested_types=[],
+  enum_types=[
```

Comment:
nit: any particular reason some of these are
```
type=[],
```
and others are
```
type=[
],
```
?

Same with extension_ranges and oneofs